CHANGELOG.md

```diff
@@ -2,6 +2,41 @@
 
 All notable changes to this project will be documented in this file. See [standard-version](https://github.com/conventional-changelog/standard-version) for commit guidelines.
 
+## [1.5.0](https://github.com/aws/aws-rfdk/compare/v1.4.0...v1.5.0) (2024-12-05)
+
+
+### Supported CDK Version
+
+* [2.163.0](https://github.com/aws/aws-cdk/releases/tag/v2.163.0)
+
+
+### Officially Supported Deadline Versions
+
+* [10.1.9.2 to 10.4.0.10](https://docs.thinkboxsoftware.com/products/deadline/10.4/1_User%20Manual/manual/release-notes.html)
+
+
+### ⚠ BREAKING CHANGES
```

Comment:
In the past, we put guidance on navigating breaking changes in the `packages/aws-rfdk/doc/upgrade` directory:

https://github.com/aws/aws-rfdk/tree/mainline/packages/aws-rfdk/docs/upgrade

We then link to that from the `CHANGELOG.md`, for example:

https://github.com/aws/aws-rfdk/blob/3130c67e61b636c1732edb4ee71eed64278de585/CHANGELOG.md?plain=1#L188-L190

Reply:
I did have a line like that, but it was a few lines down. I've moved it up to the top of the breaking changes heading.